src/utils/render-fragments.js

```diff
@@ -153,13 +148,29 @@ export const getGlossaryUsedLink = (slug, allGlossary) => {
 
 export const getGlossaryItemsUsedInRule = slug => {
```

Comment:
The logic for coming up with the glossary for a rule should not live in a render method. That stuff needs to be isolated. Quite possibly this `glossaries-in-glossaries.json` file shouldn't even exist, and all that info should be rolled up into a simple function that, you pass it the rule ID, and it returns an array of definitions IDs used in that rule.